### ISSUE TITLE
PC2D gets stuck at 45 degree angle using Polygon Collider 2D

### ISSUE THREAD

**Issue by DeanTroub** [cjddmut/Unity-2D-Platformer-Controller#74]
The player is getting stuck at the start of going up a 45 degree angle. I am using a Polygon Collider 2D to make the collider. Using Unity 2019.1.0a8.

![pc2dissue](https://user-images.githubusercontent.com/45073817/48565425-bfd60900-e8b5-11e8-86d5-df7389ea5dc8.gif)

Here are my current settings:
![pc2dissue](https://user-images.githubusercontent.com/45073817/48565949-ed6f8200-e8b6-11e8-9c9a-735254dcff88.PNG)

Edit: After further testing, this does not occur with one iteration, only 2 or more.

**Comment by jdiogobc** [comment]
Hey! Did you manage to get it smooth? Changing the iterations didn't do it for me )=

**Comment by DeanTroub** [comment]
Yeah, using the prefab that comes with it and only changing the iterations
to 1, fixed the sticking at the start of slopes. I only had 45 degree
slopes though.

On Dec 22, 2018 13:06, "jdiogobc" <notifications@github.com> wrote:

Hey! Did you manage to get it smooth? Changing the iterations didn't do it
for me )=

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/cjddmut/Unity-2D-Platformer-Controller/issues/74#issuecomment-449594823>,
or mute the thread
